internals/daemon/api_files_test.go

```diff
@@ -470,28 +472,19 @@ func (s *filesSuite) TestMakeDirsUserGroupMocked(c *C) {
 		return &u, &g, nil
 	}
 
-	var mkdirAllChownCalls []args
-	mkdirAllChown = func(path string, perm os.FileMode, uid sys.UserID, gid sys.GroupID) error {
-		mkdirAllChownCalls = append(mkdirAllChownCalls, args{path, perm, uid, gid})
-		return os.MkdirAll(path, perm)
-	}
-
 	defer func() {
-		mkdirChown = osutil.MkdirChown
+		mkdir = osutil.Mkdir
 		normalizeUidGid = osutil.NormalizeUidGid
-		mkdirAllChown = osutil.MkdirAllChown
 	}()
 
 	tmpDir := s.testMakeDirsUserGroup(c, 12, 34, "USER", "GROUP")
 
-	c.Assert(mkdirChownCalls, HasLen, 3)
-	c.Check(mkdirChownCalls[0], Equals, args{tmpDir + "/normal", 0o755, osutil.NoChown, osutil.NoChown})
-	c.Check(mkdirChownCalls[1], Equals, args{tmpDir + "/uid-gid", 0o755, 12, 34})
-	c.Check(mkdirChownCalls[2], Equals, args{tmpDir + "/user-group", 0o755, 56, 78})
-
-	c.Assert(mkdirAllChownCalls, HasLen, 2)
-	c.Check(mkdirAllChownCalls[0], Equals, args{tmpDir + "/nested1/normal", 0o755, osutil.NoChown, osutil.NoChown})
-	c.Check(mkdirAllChownCalls[1], Equals, args{tmpDir + "/nested2/user-group", 0o755, 56, 78})
+	c.Assert(mkdirCalls, HasLen, 5)
+	c.Check(mkdirCalls[0], Equals, args{tmpDir + "/normal", 0o755, osutil.MkdirOptions{}})
```

Comment:
It seems odd that we have `ExistOK: true` on all the others, but not this one. Why is it different?

Reply:
Added it back for consistency.

Reply:
Resolved in the other comment.